frontend/javascripts/types/api_flow_types.ts

```diff
@@ -702,3 +703,166 @@ export type ZarrPrivateLink = {
   accessToken: string;
   expirationDateTime: number | null;
 };
+
+export enum VoxelyticsRunState {
```

Comment:
If you want to lazy-import a vx bundle, it might be useful to add these types to a new module. In general, we should reorganize some code a bit (e.g., this module currently imports the store which imports the sagas which imports a big part of the code base). However, this is rather out of scope for this PR.

Reply:
I think it makes more sense to refactor that in a coordinated effort. If moving these types in a separate file, we at least also need to move the vx-related functions from `admin_rest_api.ts` to a separate file.